system/mp/mpcomm.cpp

```diff
@@ -1801,7 +1823,7 @@ CMPConnectThread::CMPConnectThread(CMPServer *_parent, unsigned port)
             port = minPort + getRandom() % numPorts;
             try
             {
-                listensock = ISocket::create(port, 16);  // better not to have *too* many waiting
+                listensock = ISocket::create(port, 800);
```

Comment:
I assumed you would be checking here (to see if the value you were trying to set was greater than somaxconn).

If you are running on an older kernel the value is hardwired to 128 (and defined by the macro SOMAXCONN) I think.